src/specs/utils.spec.js

```diff
@@ -0,0 +1,65 @@
+import moment from 'moment';
+import { groupByDate, getDailySummary, sortByDate } from '../utils';
+import testData from './testData.json';
+
+const { list } = testData;
+
+describe('groupByDate', () => {
+  it('returns an object of arrays', () => {
+    const result = groupByDate(list);
+    expect(typeof result).toBe('object');
+
+    const firstItem = Object.values(result)[0];
+    expect(Array.isArray(firstItem)).toBe(true);
+  });
+
+  it('returns an object of arrays grouped by date', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
  it("returns an object of arrays grouped by date", () => {
```